libs/application/templates/social-insurance-administration/old-age-pension/src/fields/Conclusion/Conclusion.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { FC } from 'react'
+import { useNavigate } from 'react-router-dom'
+import {
+  ApplicationConfigurations,
+  FieldBaseProps,
+} from '@island.is/application/types'
+import {
+  Box,
+  Bullet,
+  BulletList,
+  Button,
+  AlertMessage,
+  ActionCard,
+  AccordionCard,
+  Text,
+} from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { oldAgePensionFormMessage } from '../../lib/messages'
+import RetirementIllustration from '../../assets/Images/Retirement'
+import * as styles from './Conclusion.css'
+import Markdown from 'markdown-to-jsx'
+
+const Conclusion: FC<FieldBaseProps> = ({ application, field }) => {
```

Comment:
We created a custom component for the conclusion screen because we could not do the following:
1. Cannot change alertType in alert message (default "success")
<img width="618" alt="image" src="https://github.com/island-is/island.is/assets/54938148/46a6a818-b46c-4230-bcd2-4e9fef02365b">

2. Unable to change header text (default "Umsókn móttekin!")
3. It is not possible to add an ActionCard with a link to the Income Plan
<img width="615" alt="image" src="https://github.com/island-is/island.is/assets/54938148/860220c0-1802-48d9-a5a9-c52c54b3bb3c">

4. Cannot add AccordionCard with information on other applications for which the applicant may be eligible
<img width="615" alt="image" src="https://github.com/island-is/island.is/assets/54938148/27195ad3-a2d3-4a84-a607-365ac2712aff">

5. Unable to insert image

Reply:
I talked to @GunnarIngiReykjalin about this issue, I can't accept the screen in the current state since there is a strict enforcement that all conclusion screens need to be the same for all new applications.
He suggested that your team have a discussion with him on what your team needs from the conclusion screen and the reasoning behind the requirements. He will then get those requirements to us and we will add them asap.

Reply:
okok we will do that. Thanks!